test/org/sagebionetworks/bridge/dynamodb/DynamoStudyTest.java

```diff
@@ -53,12 +53,14 @@ public void studyFullySerializesForCaching() throws Exception {
         assertEquals(study.getMaxNumOfParticipants(), node.get("maxNumOfParticipants").asInt());
         assertEquals(study.getStormpathHref(), node.get("stormpathHref").asText());
         assertEquals(study.getPasswordPolicy(), JsonUtils.asEntity(node, "passwordPolicy", PasswordPolicy.class));
-        assertEquals(study.getVerifyEmailTemplate(), JsonUtils.asEntity(node, "verifyEmailTemplate", EmailTemplate.class));
+        assertEquals(study.getVerifyEmailTemplate(), JsonUtils.asEntity(node, "verifyEmailTemplate",
+                EmailTemplate.class));
```

Comment:
I don't know if we ever discussed line width... I'm set to 120 which seems to work even on the laptop, what are you using? What's our standard? Also I'm assuming 4 spaces per tab is our standard. We should probably write done at https://sagebionetworks.jira.com/wiki/display/BRIDGE/Code+Conventions